src/clang.rs

```diff
@@ -217,9 +217,17 @@ impl Cursor {
         if self.is_toplevel() {
             return false;
         }
+
         let parent = self.semantic_parent();
-        (parent.is_template() && !parent.is_fully_specialized_template()) ||
-        parent.is_in_non_fully_specialized_template()
+        if parent.is_fully_specialized_template() {
+            return false;
+        }
+
+        if !parent.is_template_like() {
+            return false;
```

Comment:
I suppose we should still check recursively whether parent `is_in_non_fully_specialized_template`, otherwise something like
```c++
template<typename... T>
struct Outer {
  struct Inner {
    static constexpr int value[] = { T::value, ... };
  };
};
```
may crash again?